pkg/printers/internalversion/printers.go

```diff
@@ -60,7 +59,20 @@ import (
 	"k8s.io/kubernetes/pkg/util/node"
 )
 
-const loadBalancerWidth = 16
+const (
+	loadBalancerWidth = 16
+
+	// LabelNodeRoleMaster specifies that a node is a master
+	// It's copied over to here until it's merged in core: https://github.com/kubernetes/kubernetes/pull/39112
```

Comment:
Is this a TODO? If so, I would reference this pull request (#50438) and specify exactly what has to happen for the TODO to be executed.